bentoml/_internal/io_descriptors/image.py

```diff
@@ -21,11 +19,10 @@
 
     # NOTE: pillow-simd only benefits users who want to do preprocessing
     # TODO: add options for users to choose between simd and native mode
-    _exc = const.IMPORT_ERROR_MSG.format(
```

Comment:
👍 we should probably get rid of the IMPORT_ERROR_MSG constant, an inline string like this makes it more readable

Reply:
we already removed the import_error_msg